app/controllers/authors_controller.rb

```diff
@@ -0,0 +1,74 @@
+class AuthorsController < ApplicationController
```

Comment:
I don't think there should be an `AuthorsController`.
If you don't use it please remove it.